src/seedu/addressbook/AddressBook.java

```diff
@@ -1163,12 +1176,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param stringWithPrefixSign  Parameter as a string
+     * @param fullString  Parameter as a string
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String stringWithPrefixSign, String sign) {
-        return stringWithPrefixSign.replace(sign, "");
+    private static String removePrefix(String fullString, String sign) {
+        return fullString.replace(sign, "");
```

Comment:
Good job, but you still need to change `sign` parameter. Refer to: https://github.com/nus-cs2103-AY1617S2/addressbook-level1#abstract-methods-well-lo-methodabstraction